core/lib/health_check/src/lib.rs

```diff
@@ -23,6 +30,9 @@ pub enum HealthStatus {
     Ready,
     /// Component is affected by some non-fatal issue. The component is still considered healthy.
     Affected,
```

Comment:
Affected? Very interesting. What's an example for this? Would we want to return this status when state keeper is working with Postgres storage? cc @itegulov

Reply:
Two uses of this status rn are:

- If the EN is out of sync with the main node
- If commitment mismatch was detected by the consistency checker

Just in case: this status doesn't lead to an erroneous `/health` response status.